src/sundials/sundials_utils.h

```diff
@@ -24,6 +24,9 @@
 #include <sundials/sundials_config.h>
 #include <sundials/sundials_types.h>
 
+/* width of name field in sunfprintf_<type> for aligning table output */
+#define SUN_TABLE_WIDTH 28
```

Comment:
This was sufficiently wide in the initial implementation but some new steppers have been added since then so it would be good to double check.

Reply:
The longest name is 27 characters, so it's sufficient. I suppose it could even be reduced.